internal/view/exec.go

```diff
@@ -267,7 +267,7 @@ func launchShellPod(a *App, node string) error {
 		return err
 	}
 
-	for i := 0; i < k9sShellRetryCount; i++ {
+	for i := 0; i < timeout; i++ {
```

Comment:
Not a timeout! This is really the original retryCount best to live the retryCount alone and have a settable timeout ie k9sShellRetryDelay?